rosdep/python.yaml

```
@@ -5880,6 +5880,22 @@ python3-aiohttp-cors:
   fedora: [python-aiohttp-cors]
   gentoo: [dev-python/aiohttp-cors]
   ubuntu: [python3-aiohttp-cors]
+python3-aiortc-pip:
+  debian:
+    pip:
+      packages: [aiortc]
+  fedora:
+    pip:
+      packages: [aiortc]
+  gentoo:
+    pip:
+      packages: [aiortc]
+  osx:
+    pip:
+      packages: [aiortc]
+  ubuntu:
+    pip:
+      packages: [aiortc]
```

```suggestion
python3-aiortc-pip:
  debian:
    bookworm: [aiortc]
  osx:
    pip:
      packages: [aiortc]
  ubuntu:
    jammy: [aiortc]
```

I would suggest not backfilling with pip packages. They can't be used in buildfarm-released packages so it's better to set up the definitons without pip allowing them to be used for future releases.

---

Should it be renamed to `python3-aiortc` if so?

---

Yes, I think this should be renamed to `python3-aiortc`